Editors/BackgroundEditor.ui

```diff
@@ -352,6 +355,19 @@
          </layout>
         </widget>
        </item>
+       <item row="0" column="0">
+        <widget class="QLabel" name="nameLabel">
+         <property name="text">
+          <string>&amp;Name</string>
+         </property>
+         <property name="buddy">

```

Comment:
What is "buddy"?

Reply:
The buddy is where if you press the mnemonic key of the label the buddy is focused. That's why the N is underlined in GameMaker.

![GM Label Buddy](https://user-images.githubusercontent.com/3212801/90988906-66a80880-e564-11ea-99a0-a77f98cfb47e.png)